### PR TITLE
Fix host selection variable

### PR DESCRIPTION
**Pull request by carsongee**
Switching to hostname allows this to work in the vagrant cluster as well as a real inventory.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @carsongee! I've created [OSPR-169](https://openedx.atlassian.net/browse/OSPR-169) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. Feel free to add information to the ticket which can help Product understand the context for the PR - supporting documentation, edx-code email threads, timeline information ('this must be merged by XX date', and why that is), partner information (this is for a course on edx.org, for example), etc. 

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx.readthedocs.org/projects/userdocs/en/latest/process/overview.html).

File: playbooks/roles/mariadb/tasks/cluster.yml [hunk 1]
```diff
@@ -14,12 +14,16 @@
   service: name=mysql state=stopped
   when: not mariadb_bootstrap.stat.exists
 
+- debug: var=hostvars
```
Comment (e0d) [inline playbooks/roles/mariadb/tasks/cluster.yml]: Want this merged?

**Comment by e0d** [comment]
Other than the comment about the debug lines, this LGTM.

**Comment by carsongee** [comment]
Good catch, thanks.

**Pull request merged**